### DISCUSSION TITLE
404: NOT_FOUND Code: DEPLOYMENT_NOT_FOUND

### DISCUSSION THREAD

**Discussion by baraglobal** [vercel/community#4532] [Help]
### Summary

I'm having trouble accessing my app on the default vercel domain wherever I'm not logged into Vercel. It's using the Next / Sanity Started. As far as I can tell there is a working production deployment. Am I missing something really obvious?

### Example

https://harley-weir-archive.vercel.app

### Steps to Reproduce

Visit the link!

**Comment by vitustockholm** [comment]
problems with storefront redirection, you should check your env variables, also we need more info about deployment logs. If You were using vercel cli,- try command ,,vercel --prod" to deploy your production branch(also check versel dashboard seetings which branch is configured for production.
